crates/storage/errors/src/lib.rs

```diff
@@ -9,7 +9,6 @@
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 #![cfg_attr(not(feature = "std"), no_std)]
 
-#[cfg(not(feature = "std"))]
```

Comment:
we only need `alloc` when `std` feature is enabled and we actually use the `alloc` crate

Reply:
done